sql-database/scale-pool/scale-pool.sh

```diff
@@ -1,48 +1,64 @@
 #!/bin/bash

```

Comment:
This script fails due to \r (CR) tokens. Run dos2unix or a similar tool to change CRLF to LF.

Reply:
script succeeds for me - copy from my vscode into my bash shell - so, please explain

Reply:
figured this out, but where is this doc'd?

Reply:
https://review.docs.microsoft.com/en-us/search/?search=crlf&scope=Contributor&branch=master
no results